lib/facade_converter.ts

```diff
@@ -285,11 +294,14 @@ export class FacadeConverter extends base.TranspilerBase {
   private TS_TO_DART_TYPENAMES: ts.Map<ts.Map<string>> = {
     'lib': this.stdlibTypeReplacements,
     'lib.es6': this.stdlibTypeReplacements,
-    'typings/es6-promise/es6-promise': {'Promise': 'Future'},
     'angular2/typings/es6-promise/es6-promise': {'Promise': 'Future'},
```

Comment:
It might break `angular2`, but if you can test it, then I'll do it. I just prefer smaller deltas.